saagieapi/saagie_api.py

```diff
@@ -381,488 +380,3 @@ def get_technology_name_by_id(self, technology_id: str) -> (str, str):
             return "", ""
         repo_name, tech_name = technology_label_with_repo_name[0]
         return repo_name, tech_name
-
```

Comment:
Do you plan to do a major release that brings breaking changes? If not, users will be affected by the removal of these deprecated methods and won't be warned...

Reply:
Major release planned